### PR TITLE
Resolved mentioned HTML errors in #412

### LINKED ISSUE #412: HTML errors
## Description

The website has a few HTML errors -
- `<nav>` elements do not have a `name` attribute but one is provided on ``Line 52 https://opensource.creativecommons.org/``
- `<nav>` element closes but `<div>` element in it is still open on ``Line 71 - Line 72 https://opensource.creativecommons.org/``
- `align` attribute is no longer supported for `div` elements in `HTML5`, must use `CSS` instead. ``Line 96, 102, etc. https://opensource.creativecommons.org/``

## Reproduction

Bug is not visible on the page, but affects the code.

## Expectation
Since the bug is not visible on the page, it still displays what it should but the code isn't right

## Resolution

- [x] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by TrustedMercury**
## Fixes

Fixes #412  by @TrustedMercury

## Description

Fixes the following HTML errors - 
- Removes the `name` attribute from the `<nav>` element as it's deprecated in HTML5
- Removed the `align` attribute from the `<div>` elements as it's deprecated in HTML5
- Fixed unclosed `<div>` element inside `<nav>`

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: content/contents.lr [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Open Source at CC
...
+**[Creative Commons](https://creativecommons.org/)** has been building free software for over a decade to support our mission of maximizing digital creativity, sharing, and innovation. We work on products that, among other things, make applying CC licenses to content easier, help users find CC licensed content easily, promote and educate users about CC licenses and copyright, and connect the global CC community. Our projects come in all sizes and shapes, from several simple static websites to making the 1.6 billion+ Creative Commons works currently available on the internet [searchable](https://search.creativecommons.org/).
```
Comment (dhruvkb) [inline content/contents.lr:7]: I'm having a hard time identifying what the change is here.

File: content/contents.lr [hunk 2]
```diff
@@ -16,7 +16,7 @@ So if you're looking to integrate CC licenses or CC licensed works into your app
     <div class="card-img-top">
       <img class="mx-auto d-block" src="/comments-solid.png" alt="Community icon"/>
     </div>
-    <div class="card-body" align="center">
+    <div class="card-body">
```
Comment (dhruvkb) [inline content/contents.lr:19]: If you remove the `align` attribute, you have to add the corresponding CSS rule.

File: content/contents.lr (hunk 1 above)
Comment (TrustedMercury) [reply content/contents.lr:7]: Yeah, my editor did that for no reason and I wasn't able to resolve it. I used text difference checkers and there was no change so I didn't think it was a problem

File: content/contents.lr (hunk 2 above)
Comment (TrustedMercury) [reply content/contents.lr:19]: Will do.

Comment (TrustedMercury) [reply content/contents.lr]: ✅ Done

**Pull request merged**